Jenkinsfile

```diff
@@ -25,6 +25,12 @@ dockerNode(image: 'uf-mil:subjugator') {
 	}
 	stage("Format") {
 		sh '''
+			if [[ ! -z "$(python2.7 -m flake8 --max-line-length=120 --exclude=__init__.py .)" ]]; then
```

Comment:
I think we should  ```--ignore E731```, which prevents us from using lambdas for callbacks which I think everyone in the lab likes.